src/cryptenroll/cryptenroll-empty.c

```diff
@@ -0,0 +1,61 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "cryptenroll-empty.h"
+#include "json.h"
+
+int enroll_empty(
+                struct crypt_device *cd,
+                const void *volume_key,
+                size_t volume_key_size) {
+        _cleanup_(json_variant_unrefp) JsonVariant *v = NULL;
+        const char *node;
+        int keyslot, r, q;
+        char keyslot_str[DECIMAL_STR_MAX(keyslot)];
+
+        assert_se(cd);
+        assert_se(volume_key);
+        assert_se(volume_key_size > 0);
```

Comment:
Why `_se`?

Reply:
I copied this from here: https://github.com/systemd/systemd/blob/main/src/cryptenroll/cryptenroll-recovery.c#L24

Reply:
Please change it to `assert`, in both places.